engine/termops.ml

```diff
@@ -189,7 +189,7 @@ let pr_decl (decl,ok) =
 
 let pr_evar_source = function
   | Evar_kinds.NamedHole id -> Id.print id
-  | Evar_kinds.QuestionMark _ -> str "underscore"
+  | Evar_kinds.QuestionMark _ -> str "underscore termops"
```

Comment:
Just saw that: what's the reason for that diff? Is it a spurious change? In any case you should probably revert this line.

Reply:
Hm, yes, this seems spurious, let me change this.